web/pandas_web.py

```diff
@@ -223,9 +225,20 @@ def home_add_releases(context):
         with open(pathlib.Path(context["target_path"]) / "releases.json", "w") as f:
             json.dump(releases, f, default=datetime.datetime.isoformat)
 
-        for release in releases:
+        non_obsolete_releases = []
+
+        # This is necessary for the versions to be properly grouped
+        releases = sorted(releases, key=lambda release:version.parse(release["tag_name"]), reverse=True)
```

Comment:
I think it's better to parse all versions after loading them from json, and keep the list with the version objects instead of strings. Otherwise you need to parse 3 times, which is not very optimal and will make the script slower.